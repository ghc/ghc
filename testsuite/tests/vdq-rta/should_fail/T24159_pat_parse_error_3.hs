{-# LANGUAGE RequiredTypeArguments, LinearTypes, UnicodeSyntax #-}
module T24159_pat_parse_error_3 where

f (a ‚ä∏ b) = ()
