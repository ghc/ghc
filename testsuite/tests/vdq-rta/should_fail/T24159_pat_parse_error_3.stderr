T24159_pat_parse_error_3.hs:4:4: error: [GHC-32181]
    Illegal function arrow in pattern: ‘a %1 -> b’
    Type syntax in patterns isn't supported at the time

